var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -149,6 +149,7 @@ class Acts(CMakePackage, CudaPackage):
     depends_on('pythia8', when='+pythia8')
     depends_on('root @6.10: cxxstd=14', when='+tgeo @:0.8.0')
     depends_on('root @6.20: cxxstd=17', when='+tgeo @0.8.1:')
+    depends_on('vecmem')
```

Comment:
```suggestion
    depends_on('vecmem@0.4: +sycl', when='+sycl')
```
Not specifying a version means "we assume this works for any version", which seems overly optimistic. Let's rather say that this should work when using any vecmem later than the bundled one, which is 0.4.

Also, vecmem is currently only used by the sycl plugin. Which means that we probably want to have the SYCL plugin supported before working on adding vecmem support. [I have an experimental branch that aims to address this](https://github.com/spack/spack/compare/develop...HadrienG2:acts-fixes), but its build is currently failing somewhere during the hipsycl build, and I likely won't have time to investigate until December...